src/caret_analyze/architecture/architecture.py

```diff
@@ -207,6 +207,33 @@ def _verify(nodes: Collection[NodeStruct]) -> None:
                      f'callback_type: {uniqueness_violated[0]}'
                      f'period_ns: {uniqueness_violated[1]}'))
 
+    def assign_message_context(self, node_name: str, context_type, sub_topic_name: str, pub_topic_name: str):
+        try:
+            node: NodeStruct = Util.find_one(lambda x: x.node_name == node_name, self._nodes)
+            node.assign_message_context(context_type, sub_topic_name, pub_topic_name)
+        except ItemNotFoundError:
+            msg = 'Failed to find node. '
+            msg += f'node_name: {node_name}'
+            raise ItemNotFoundError(msg)
```

Comment:
I would like to refactor the ItemNotFoundError exception message in the future, so simplicity is fine here.
```suggestion
        node: NodeStruct = Util.find_one(lambda x: x.node_name == node_name, self._nodes)
        node.assign_message_context(context_type, sub_topic_name, pub_topic_name)
```

Reply:
Fixed it in [50b4698](https://github.com/tier4/CARET_analyze/pull/196/commits/50b4698dffa5b3c7368e101c3185d1f8b5e47ef0)